ADA/databricks_workflow_script_rstudio.qmd

```diff
@@ -0,0 +1,238 @@
+---
+title: "Scripting Workflows in RStudio"
+---
+
+For a pipeline to be built there must be scripts, queries or notebooks available to read by Databricks, either located in your workspace, or in a Git repository.
+
+For this example we will create a folder in our workspace, create two test notebooks to comprise the workflow, and then switch to RStudio to script the job and set it off running. We'll also set it up to notify us by email when the workflow successfully completes.
```

Comment:
As with the other page - worth mentioning that there's a point and click alternative to this (although I'm glad we have documented all the methods!) 

When would it be preferable to script a workflow instead of using the interface?

Reply:
As above